### DISCUSSION TITLE
How to specify a Golang version

### DISCUSSION THREAD

**Discussion by donghyunlee00** [github/securitylab#186] [General] [answered]
Hello.

I tried to create a CodeQL database for kubernetes-1.9.0 (which is an old version) through LGTM.com.

It was built successfully, but I found a warning in the Golang 'Extraction' log, as follows.

```
...
[build] Detected go version: go version go1.15 linux/amd64.
[build] Kubernetes requires go1.9.1 or greater.
[build] Please install go1.9.1 or later.
...
```

As far as I know, I can specify the version through `lgtm.yml` for languages such as Python, C#, Java.

However, in [this document](https://lgtm.com/help/lgtm/lgtm.yml-configuration-file), I could not find any words to specify the version of Golang.

Is there any way to specify the Golang version when creating a CodeQL database?

My LGTM project is https://lgtm.com/projects/g/donghyunlee00/v2-kubernetes-1.9.0, exactly same code with [kubernetes-1.9.0 release code](https://github.com/kubernetes/kubernetes/releases/tag/v1.9.0)

Best Regards.

**Comment by smowton** [comment, marked as answer]
Answered at https://github.com/github/codeql-go/issues/351
